sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultJcaProvider.java

```diff
@@ -40,7 +40,7 @@
     /**
      * Stores the version.
      */
-    private static final Double VERSION = 1.0;
```

Comment:
Made this change so we wouldn't use a deprecated API in line 49.

Reply:
It's deprecated in Java 11. But it's not deprecated in Java 8, we need to support Java 8. So please revert this line. Other wise the pipeline will failed: https://dev.azure.com/azure-sdk/public/_build/results?buildId=1075308&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=b5f103f3-7317-514c-e60c-6e19642365f6&l=17